import-export-clausewitz/pdx_data.py

```diff
@@ -359,7 +376,7 @@ def get_binary_data(self):
 
         result.extend(struct.pack("2s", b'[['))
         result.extend(struct.pack(str(len(self.name)) + "sb", self.name.encode('UTF-8'), 0))
-        result.extend(struct.pack("cb2sifff", b'!', 1, b'pf', 3, self.pos[0], self.pos[1], self.pos[2]))
+        result.extend(struct.pack("cb2sifff", b'!', 1, b'pf', 3, -self.pos[0], self.pos[2], self.pos[1]))
```

Comment:
Könntest du des vlt noch umbauen dass die Matrix benutzt wird :)
Dann sind die nämlich auch richtig wenn man noch andere transformationen in blender macht